### DISCUSSION TITLE
Agent meeting is, today, Wednesday, the 14th at 2pm MTN time.

### DISCUSSION THREAD

**Discussion by lin-fred** [ArctosDB/arctos#6405] [Agent Committee]
Hey everyone,

The next Agent meeting is, today, Wednesday, the 14th at 2pm MTN time. You can see the [Arctos calendar](https://calendar.google.com/calendar/u/2?cid=amt2OWY4a2UwYXBxamJzOG52c3FybTRyODRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) for details and zoom link. Please contact me if you have issues using the zoom information.

Take a look at the agents project page https://github.com/orgs/ArctosDB/projects/16 under the Agent Committee Discussions. Look over the issues, we will be going over them in order from top to bottom. Please add any that you want to discuss!

[Agenda](https://docs.google.com/document/d/1-nssmG4VRJFP8Ek1wPDzkfpDoqLMs3eR/edit) has been moved to a new shared google drive, if you cannot access the agenda we can fix it during the meeting.

Also, Github is getting rid of the original "teams" discussion page, and so it has now been moved to the discussions in general of the Arctos repo. Any feedback is welcome. https://github.com/orgs/ArctosDB/teams/agents-committee

Lindsey

**Comment by pwimbo** [comment]
Hi Lindsey,
I'd like to join this group, but when I click the link I get a 404.  I have
a Github account and have signed in to Arctos.  I'm not going to be able to
make the meeting today, but am interested in joining in the future.
Thanks,
Peter

**Reply by lin-fred** [reply]
Hi Peter thanks for letting me know. It looks like you aren't in the Arctos Repo yet either, so we will work on getting permissions for both that and then into the Agent team itself.

I also got your request to see the Agenda, which is in google docs, so I will be sure to add you to have access.

Be on the lookout for Google and Github invites throughout this next week.

**Comment by pwimbo** [comment]
Thanks Lindsey!
Best,
Peter
